.gitignore

```diff
@@ -1,6 +1,5 @@
-node_modules/
-
 # misc
 .DS_Store
 npm-debug.log*
-.env
\ No newline at end of file
+.env
+/api/node_modules
```

Comment:
This will behave exactly the same, for what its worth

Reply:
Im going to create a ticket to sort out our .gitignore problem, we have 3.